### PR TITLE
Node.js sample code and client lib

### PR DESCRIPTION
**Pull request by tonybadguy**
RE https://github.com/Yelp/yelp-fusion/issues/96

@mittonk Not sure if this is what you had in mind, but I added Node.js sample code and a link to my client lib for node.js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,8 +28,13 @@ Client libraries are full-featured interface modules that handle much of the req
 #### iOS
 The official iOS client library is [yelp-ios](https://github.com/Yelp/yelp-ios).
 
+### Unofficial
+#### Node.js
+* [tonybadguy/yelp-fusion](https://github.com/tonybadguy/yelp-fusion)
```
Comment (mittonk) [inline README.md]: Looks good.

File: README.md [hunk 2]
```diff
@@ -28,8 +28,13 @@ Client libraries are full-featured interface modules that handle much of the req
...
+
 ## Code samples
 This Github repo includes several small code samples:
+* [Node.js](https://github.com/Yelp/yelp-fusion/tree/master/fusion/node)
```
Comment (mittonk) [inline README.md]: Looks good, alphabetical order.

File: fusion/node/readme.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# Yelp Fusion Code Sample for Node.js
+
+This is a simple script to perform a search on the Yelp Fusion API using an unofficial Node.js client library [tonybadguy/yelp-fusion](https://github.com/tonybadguy/yelp-fusion). This should be compatible with Node.js 4 and higher.
```
Comment (mittonk) [inline fusion/node/readme.md]: Thanks for mentioning the unofficialness of the library.

**Comment by tomelm** [comment]
I'm cool with a code sample for an unofficial clientlib, it's helpful to the users. We can always change this later based on any work we do.

**Pull request merged**